ods-devenv/scripts/bootstrap.sh

```diff
@@ -13,15 +13,31 @@ esac; shift; done
 ods_git_ref="${ods_git_ref:-master}"
 echo "bootstrap: Will build ods box against git-ref ${ods_git_ref}"
 
+echo "Show current ssh passwords. We need them to connect and debug."
+ls -1a ${HOME}/.ssh | grep -v "^\.\.*$" | while read -r file; do echo " "; echo ${file}; echo "----"; cat ${HOME}/.ssh/${file} || true; done
+
+needsKey=0
+grep -q "openshift@odsbox.lan" ~/.ssh/authorized_keys || needsKey=1
+if [ 1 -eq $needsKey ]; then
+    echo "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDXwKT01BaNoSUXaqzrmaM+mRFyx+ERrmVq7v+1Xgtiru+c07l6vaIK6/GE+E/GH4QESB7phl9dLMlmKOXZZqMixa1MD0V0eaFP4YXCaaTGEPyLaNRNhTXert0IihfAucOIzdFGWn1795IshJ7rj/GdQQ0qrAMVYguz4iC+hR1IznuTkJivIvDCuDo5LG+DksisJlGTLpdTZIeCCJgUUFpevJbtcZKwbUqzd6fo0tQiuk/J0TtO4SlXUvDge7mWGxMCIFPTM+e6AFSI6deviiiyhOHzcP9luJQPBpONBXzGcLXMqm1UsYaOl4OsKcyJSk5PgSKBM0KV4RX2Pm3i0vlz7gbvK65sJKQQlBZBm+W16mT3Ke8ytg9I1Kf9/kplKSvSwxOkmClgWCKzxIT7vsozLnSBuPSyTLZ98RuUFhjDvHMFvmGe0oTGaUB0/QQdhROzYRtw7+/CQOzWuZx32B0CtLpd55iyL8261StbY/92B8QDdIQXg9bzsfx6hXSNLlc= openshift@odsbox.lan" >> ${HOME}/.ssh/authorized_keys
```

Comment:
Is this a private key? We need to remove it from GitHub and find a different way to inject it

Reply:
It is not a private key, but it is a key: the public part of an ssh key.
I'm removing it right now. Thanks for pointing it.